addons/binding/org.openhab.binding.lutron/src/main/java/org/openhab/binding/lutron/handler/TabletopKeypadHandler.java

```diff
@@ -0,0 +1,176 @@
+        LED9(89, "led9", "LED 9"),
+        LED10(90, "led10", "LED 10"),
+        LED11(91, "led11", "LED 11"),
+        LED12(92, "led12", "LED 12"),
+        LED13(93, "led13", "LED 13"),
+        LED14(94, "led14", "LED 14"),
+        LED15(95, "led15", "LED 15");
+
+        private final int id;
+        private final String channel;
+        private final String description;
+
+        Component(int id, String channel, String description) {
+            this.id = id;
+            this.channel = channel;
+            this.description = description;
+        }
+
+        @Override
+        public int id() {
+            return id;
+        }
+
+        @Override
+        public String channel() {
+            return channel;
+        }
+
+        @Override
+        public String description() {
+            return description;
+        }
+
+    }
+
+    private static final List<Component> buttonGroup1 = Arrays.asList(Component.BUTTON1, Component.BUTTON2,
+            Component.BUTTON3, Component.BUTTON4, Component.BUTTON5);
+    private static final List<Component> buttonGroup2 = Arrays.asList(Component.BUTTON6, Component.BUTTON7,
+            Component.BUTTON8, Component.BUTTON9, Component.BUTTON10);
+    private static final List<Component> buttonGroup3 = Arrays.asList(Component.BUTTON11, Component.BUTTON12,
+            Component.BUTTON13, Component.BUTTON14, Component.BUTTON15);
+
+    private static final List<Component> buttonsBottomRL = Arrays.asList(Component.BUTTON16, Component.BUTTON17,
+            Component.LOWER3, Component.RAISE3);
+    private static final List<Component> buttonsBottomCRL = Arrays.asList(Component.LOWER1, Component.RAISE1,
+            Component.LOWER2, Component.RAISE2, Component.LOWER3, Component.RAISE3);
+
+    private static final List<Component> ledGroup1 = Arrays.asList(Component.LED1, Component.LED2, Component.LED3,
+            Component.LED4, Component.LED5);
+    private static final List<Component> ledGroup2 = Arrays.asList(Component.LED6, Component.LED7, Component.LED8,
+            Component.LED9, Component.LED10);
+    private static final List<Component> ledGroup3 = Arrays.asList(Component.LED11, Component.LED12, Component.LED13,
+            Component.LED14, Component.LED15);
+
+    private final Logger logger = LoggerFactory.getLogger(TabletopKeypadHandler.class);
+
+    @Override
+    protected boolean isLed(int id) {
+        return (id >= 81 && id <= 95);
+    }
+
+    @Override
+    protected boolean isButton(int id) {
+        return (id >= 1 && id <= 25);
+    }
+
+    @Override
+    protected boolean isCCI(int id) {
+        return false;
+    }
+
+    @Override
+    protected void configureComponents(String model) {
+        String mod = model == null ? "null" : model;
+        logger.debug("Configuring components for keypad model {}", model);
+
+        switch (mod) {
+            default:
+                logger.warn("No valid keypad model defined ({}). Assuming model T15RL.", mod);
```

Comment:
If this is an optional configuration parameter it is by design so it should be logged on debug instead.